<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="random_goal" default="false" />
  <arg name="turtlebot_name"  value="$(optenv TURTLEBOT_NAME unknown_node)" />

  <!--
  <node name="random_goals_publisher"
	pkg="turtlebot_control"
	type="random_goals_publisher">
    <env name="RANDOM_GOAL" value="$(arg random_goal)" />
  </node>
  -->

  <node name="npc_node_parser" pkg="turtlebot_control" type="npc_node_parser">
    <remap from="flag_from_master" to="$(arg turtlebot_name)/flag_from_master"/>
  </node>

  <include file="$(find rct_turtlebot)/launch/amcl_nav.launch" >
    <arg name="turtlebot_name" value="$(arg turtlebot_name)" /> 
  </include>

</launch>
